rolling/distribution.yaml

```diff
@@ -1018,27 +1018,6 @@ repositories:
       url: https://github.com/ros/diagnostics.git
       version: ros2
     status: maintained
-  dolly:
-    doc:
-      type: git
-      url: https://github.com/chapulina/dolly.git
-      version: galactic
-    release:
-      packages:
-      - dolly
-      - dolly_follow
-      - dolly_gazebo
```

Comment:
I'm not 100% sure of this, but is it only `dolly_gazebo` that is failing here?  If that is the case, maybe we should just blacklist that particular package, rather than totally removing the project.